### ISSUE TITLE
no parser could be inferred

### ISSUE THREAD

**Issue by pbergner** [dangmai/prettier-plugin-apex#202]
I'm trying to configure this in a legacy org with the old `src` main folder instead of `force-app`. I'm also not using VSCode, not sure if that info is relevant.

I've tried running `sudo prettier --debug-check "**/*.{trigger,cls}"`

And its iterating over all of my classes and triggers but I'm getting an error for each one:
`src/triggers/User_Trigger.trigger[error] No parser could be inferred for file: src/triggers/User_Trigger.trigger`

My `.prettierrc` file is simply:
`{
  "trailingComma": "all"
}`

Any ideas?

**Comment by dangmai** [comment]
Thank you for the bug report, that is indeed really weird and shouldn't be happening. I built this as a CLI application first, so VSCode is definitely not necessary to use it. May I ask what OS you're using, and what version of Prettier and Prettier Apex?

**Comment by pbergner** [comment]
I was actually just able to resolve it by installing and running globally. 

MacOS : Catalina 10.15.3
prettier: 2.0.4
plugin: prettier-plugin-apex@1.3.0

Thanks for the quick reply though!

**Comment by dangmai** [comment]
Cool, I'm glad you got it working!

**Comment by kratoon** [comment]
I started to face this issue too when upgraded to the newest prettier and prettier-plugin-apex version.

I don't want prettier to be installed globally, and **managed to fix it** by adding "plugins" in the .prettierrc.
```
"plugins": ["prettier-plugin-apex"]
``` 


Issue first happened when I upgraded:
"prettier": "^2.7.1" => "^3.2.5"
"prettier-plugin-apex": "^1.10.0" => "^2.1.4"

I'm using nvm, not sure if that could be related.
